src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method `getStatistic` is quite large and handles multiple responsibilities, such as reading from a file, processing data, and writing to a file. According to the checklist, methods should be short and responsible for one task. Consider breaking this method into smaller, more focused methods.